UnityProject/Assets/Scripts/Objects/TeslaEnergyBall.cs

```diff
@@ -335,7 +335,7 @@ private void OnPulse(ElectricalArc arc)
 			}
 			else if (arc.Settings.endObject.TryGetComponent<Integrity>(out var integrity) && integrity != null && integrity.Resistances.LightningDamageProof == false)
 			{
-				integrity.ApplyDamage(damage * ((int)currentStage + 1), AttackType.Magic, DamageType.Burn, explodeOnDestroy: true);
+				integrity.ApplyDamage(damage * ((int)currentStage + 1), AttackType.Magic, DamageType.Burn, true, explodeOnDestroy: true);
```

Comment:
tesla does magic damage?

Reply:
i copied and changed around nooneys lightning spell, not sure what damage it does in tg

Reply:
ya, good enough for now. We can always change it if we need it.